README.md

```diff
@@ -12,7 +12,8 @@ Quick start examples for integrating [Banuba SDK on Android](https://docs.banuba
 3. Copy `aar` and `include` files from the Banuba SDK archive into `libs` dir:
     `BNBEffectPlayer/include` => `quickstart-android-c-api/app/libs/include`
     `BNBEffectPlayer/banuba_effect_player_c_api-release.aar` => `quickstart-android-c-api/app/libs/banuba_effect_player_c_api-release.aar`
-4. Copy and Paste your client token into appropriate section of `com/banuba/sdk/example/common/BanubaClientToken.kt`
+4. Copy and Paste your client token into appropriate section of `com/banuba/quickstart_c_api/BanubaClientToken.java`
+5. Configure effect in the appropriate section of `com/banuba/quickstart_c_api/MainActivity.java`, e.g. effects/test_BG The effects folder should be located in the `app/src/main/assets/` folder. More effects can be found [here](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters)
```

Comment:
Please make sure that you have .gitkeep file to let the folder `app/src/main/assets/` exist as a part of repo. Also changes in this folder should be ignored by git.